go13.go

```diff
@@ -0,0 +1,7 @@
+// +build !go1.4
```

Comment:
For reference, I used the [`exact`](https://github.com/golang/tools/tree/1afe783e9f23b46389ab7a439ed4e0eee558382a/go/exact) package as reference for the naming/style of a Go 1.4+ and earlier solution.